src/containers/Rating/index.js

```diff
@@ -0,0 +1,105 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import StarRatings from 'react-star-ratings';
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+import { Button, Modal } from 'semantic-ui-react';
+import {
+  addRating, fetchRating,
+} from '../../redux/actions';
+import './style.sass';
+import { isLoggedIn } from '../../utils/auth';
+import { isOwner} from "../../utils/permissions";
+import { Dimmer, Loader, Segment } from 'semantic-ui-react'
+
+
+class Rating extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      open: false,
+    };
+  }
+
+  show = size => () => { 
```

Comment:
i think this method can be given a better name to describe what it is showing i.e if its rating then `showRating` can be ideal. same with the `close` method

Reply:
I've renamed the functions appropriately